src/account-info/ProfileScreen.js

```diff
@@ -129,7 +129,7 @@ export default function ProfileScreen(props: Props): Node {
   return (
     <SafeAreaView mode="padding" edges={['top']} style={{ flex: 1 }}>
       <ScrollView>
-        <AccountDetails user={ownUser} />
+        <AccountDetails user={ownUser} displayUserEmail={false} />
```

Comment:
Comment added.  I think this is better as a prop because the `AccountDetails` never renders the email if it is a child of `ProfileScreen`.  i.e. it can't just use `state.realm.emailAddressVisibility` to decide whether to render email or not, it also needs to know if it's in `ProfileScreen` or in `AccountDetailsScreen`.

Reply:
> Comment added.

Here's that comment:

```js
{/* displayUserEmail is false because email is not displayed when AccountDetails is rendered in ProfileScreen */}
```

It makes sense that passing `false` for a prop named `displayUserEmail` would cause the user's email not to be displayed; it would be odd if it didn't. 🙂 So we don't need a comment explaining that.

The surprising thing I'm hoping to understand is _why_ this caller doesn't want to show the user's email. Was there some discussion on that that I missed? I notice that the web app still shows the email address when viewing the self-user's details.

Reply:
I can make it conditionally render in profileScreen if that’s what you want. The profile screen in the mobile app doesn’t display the users local time or activity level and so I figured keeping it decluttered by also not showing the email made sense. I did include a screen shot of profile screen above with the email hidden but I didn’t include a comparison with it showing.

If I’m looking at the right region of the web app you’re saying shows the email, it also shows the users local time too which is one of the things I noted above is currently hidden in mobile on profile screen.

Reply:
(I've continued this out in the top-level thread, here: https://github.com/zulip/zulip-mobile/pull/5515#issuecomment-1281797489 )